src/domain/service/note.ts

```diff
@@ -117,8 +141,21 @@ export default class NoteService {
    * @param id - note internal id
    * @param content - new content
    * @param noteTools - tools which are used in note
+   * @param userId - id of the user that made changes
    */
-  public async updateNoteContentAndToolsById(id: NoteInternalId, content: Note['content'], noteTools: Note['tools']): Promise<Note> {
+  public async updateNoteContentAndToolsById(id: NoteInternalId, content: Note['content'], noteTools: Note['tools'], userId: User['id']): Promise<Note> {
+    /**
+     * If content changes are valuable, they will be saved to note history
+     */
+    if (await this.checkContentDifference(id, content)) {
```

Comment:
Maybe smth like this?

```suggestion
    if (await this.areChangesSignificant(id, content)) {
```

Otherwise it's not clear what we are checking content difference for